eng/tools/openapi-diff-runner/test/command-helpers.test.ts

```diff
@@ -206,17 +204,13 @@ describe("command-helpers", () => {
   const mockReadFileSync = vi.mocked(readFileSync);
   const mockWriteFileSync = vi.mocked(writeFileSync);
   const mockRmSync = vi.mocked(rmSync);
-  const mockPath = vi.mocked(path);
   const mockFileURLToPath = vi.mocked(fileURLToPath);
   const mockGetChangedFilesStatuses = vi.mocked(getChangedFilesStatuses);
 
   beforeEach(() => {
     vi.clearAllMocks();
 
     // Setup default mocks
-    mockPath.resolve.mockImplementation((...paths) => paths.join("/"));
```

Comment:
These mocks broke tests in this PR on Windows, and I think they should be unnecessary, because the runtime implementation should work even in tests, on all OS.